.github/workflows/script.sh

```diff
@@ -20,11 +20,11 @@ source ./.github/settings.sh
 
 case "$MODE" in
 test)
-    bazel test -c opt $BAZEL_OPTS $BAZEL_CXXOPTS //...
```

Comment:
Did you mean to drop `$BAZEL_CXXOPTS`?  If so, why?

Reply:
Yes, BAZEL seems to read `BAZEL_CXXOPTS` directly.